projects/1.20/assets/veggies-delight/veggiesdelight/lang/zh_cn.json

```
@@ -52,64 +52,64 @@
   "item.veggiesdelight.wheat_pouch": "袋装小麦种子",
 
   "item.veggiesdelight.beetroot_brownie": "甜菜根布朗尼切片",
-  "item.veggiesdelight.carrot_juice": "胡萝卜汁",
-  "item.veggiesdelight.dandelion_juice": "蒲公英汁",
+  "item.veggiesdelight.carrot_juice": "奶香胡萝卜汁",
+  "item.veggiesdelight.dandelion_juice": "奶香蒲公英汁",
   "item.veggiesdelight.dandelion_leaf": "蒲公英叶",
 
-  "block.veggiesdelight.vegan_pizza": "全素披萨",
+  "block.veggiesdelight.vegan_pizza": "纯素披萨",
   "block.veggiesdelight.sweet_potato_pie": "红薯派",
   "block.veggiesdelight.beetroot_brownie_tray": "甜菜根布朗尼",
 
-  "block.veggiesdelight.mature_dandelion": "成熟的蒲公英",
-  "block.veggiesdelight.wild_bellpeppers": "野生甜椒",
-  "block.veggiesdelight.wild_cauliflowers": "野生花椰菜",
+  "block.veggiesdelight.mature_dandelion": "成熟蒲公英",
+  "block.veggiesdelight.wild_bellpeppers": "野生甜椒丛",
+  "block.veggiesdelight.wild_cauliflowers": "野生花椰菜丛",
   "block.veggiesdelight.wild_garlic": "野生大蒜",
   "block.veggiesdelight.wild_sweet_potatoes": "野生红薯",
 
   "creativetab.veggiesdelight_tab": "蔬菜乐事",
 
   "veggiesdelight.advancement.root": "蔬菜乐事",
-  "veggiesdelight.advancement.root.desc": "去采摘你的蔬菜！",
+  "veggiesdelight.advancement.root.desc": "快去获取你的蔬菜吧！",
 
-  "veggiesdelight.advancement.fruit": "这不是水果吗？",
+  "veggiesdelight.advancement.fruit": "这难道不是水果吗？",
   "veggiesdelight.advancement.fruit.desc": "我们中出了一个叛徒",
 
-  "veggiesdelight.advancement.first": "素来如此",
-  "veggiesdelight.advancement.first.desc": "收集你的第一个蔬菜",
+  "veggiesdelight.advancement.first": "生菜起步",
+  "veggiesdelight.advancement.first.desc": "收获你的第一份蔬菜",
 
-  "veggiesdelight.advancement.all": "Gotta catch 'em all",
-  "veggiesdelight.advancement.all.desc": "收集所有蔬菜",
+  "veggiesdelight.advancement.all": "收集全餐",

```

```suggestion
  "veggiesdelight.advancement.all": "把他们全收集！",
```
宝可梦梗，https://wiki.52poke.com/wiki/Gotta_Catch_Em_All

因为是乐曲名，所以可以不翻

---

OK（这个模组怎么这么多梗？？？）